src/theme/DocItem/Footer/styles.module.css

```diff
@@ -0,0 +1,93 @@
+.lastUpdated {
+  margin-top: 0.2rem;
+  font-style: italic;
+  font-size: smaller;
+}
+
+@media (min-width: 997px) {
+  .lastUpdated {
+    text-align: right;
+  }
+}
+
+.feedback_question {
+  font-size: 14px;
+  font-style: italic;
+  padding-right: 14px;
+  line-height: 40px;
+}
+
+.thanks {
+  margin-left: 0px;
+  font-size: 14px;
```

Comment:
There are a few font sizes used in this file, and all are inline. Could we use a variable?

Also, I would try to make this more uniform:
<img width="964" alt="image" src="https://user-images.githubusercontent.com/2670454/223180974-65ffc03f-354a-41e8-8bae-14fc3094fa39.png">

I think the feedback stuff could use the same font size as the “Edit this page” link.

Reply:
And the horizontal/vertical margins should be the same.